R/my_first_fcn.R

```diff
@@ -0,0 +1,26 @@
+#' Derive on boarding dummy function
+#'
+#' @author   Fanny Gautier
+#'
+# Mandatory fields in header:
+#' @param    first_time TRUE or FALSE
+#' @details  outputs "Welcome to the admiral family!".
+#' @return   a Welcome message
+#' @export
+#' @keywords utils_print
+#' @family   utils_print
+#  keywords  der_message is not a known tag, so not included for this function
+#  family    der_message is not a known tag, so not included for this function
+#' @examples
+#' library(testthat)
```

Comment:
how come you are doing `library(testthat)`  ????

Oh I see you are using expect function ummm...I wonder if there is something else we could use here??